blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -206,4 +268,28 @@ protected MetricData getHistogramsByGranularity(String tenantId,
 
         return data;
     }
+
+    private class UnitsGetter implements Callable {
```

Comment:
Its totally up to you, but `UnitsFetcher` seems nicer and prettier than `UnitsGetter`?